### PR TITLE
Fixes #269: Removed unwanted code from the collegeElections App

### LINKED ISSUE #269: Remove unwanted code from collegeElections app
- Issue type: Bug report

The collegeElections app has so much of unwanted code. Remove the unnecessary code from the files.

### PR DESCRIPTION
**Pull request by kavithaenair**
### Short description
Fixes #269 

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: collegeElections/index.html [hunk 1]
```diff
@@ -13,69 +13,19 @@
...
-              background-color: ;
-            }
+        .serviceBox1, .serviceBox3 {
+          float:left;
+          width:30%;
+          height: 380px;
+          background-color: ;
+        }
         .serviceBox2 {
               float: left;
```
Comment (djmgit) [inline collegeElections/index.html]: Can you please fix the indentation problem here as a part of this PR?

File: collegeElections/index.html [hunk 2]
```diff
@@ -189,11 +139,8 @@
 
                  $http.jsonp(QueryCommand1).then(function(response) {
                     $scope.candidateReq.one = response.data
-                    //console.log('one');
                     $http.jsonp(QueryCommand2).then(function(response) {
                       $scope.candidateReq.two = response.data
```
Comment (djmgit) [inline collegeElections/index.html]: here too

Comment (kavithaenair) [inline collegeElections/index.html]: sure @djmgit

File: collegeElections/index.html (hunk 1 above)
Comment (kavithaenair) [inline collegeElections/index.html]: 👍

File: collegeElections/index.html (hunk 2 above)
Comment (kavithaenair) [inline collegeElections/index.html]: @djmgit I can't find any indentation problem here

Comment (djmgit) [inline collegeElections/index.html]: @kavithaenair my bad, there is no indentation problem here, the PR is good to go now.

**Pull request merged**